xkbcommon/xkbcommon.h

```diff
@@ -941,6 +941,18 @@ xkb_keymap_key_for_each(struct xkb_keymap *keymap, xkb_keymap_key_iter_t iter,
                         void *data);
 
 /**
+ * Find the keycode for a given keyname in a keymap.
+ *
+ * If no corresponding key is found in the keymap, returns 0.
```

Comment:
Should say `XKB_KEYCODE_INVALID`.

Also, please write it like so, again for consistency:

```
@returns The keycode.  If no key with this name exists, returns
XKB_KEYCODE_INVALID.
```